commitizen-tools/commitizen
# Add custom tag message

## Description
According to #558, I implemented a custom tag message for annotated tags.
Right now, the annotated tag is given as "tag_name" with its message as "tag_name" as well. It might be nice to have custom tag messages, for example, to add the features added on that tag.
The idea is a new argument to bump with the tag_message,

```bash
cz bump --annotated-tag-message "a message"
cz bump -atm "a message"
```

## Checklist


## Expected behavior
Running either the following commands will create an annotated tag with "a message" as the message of the tag.

```bash
cz bump --annotated-tag-message "a message"
cz bump -atm "a message"
```

## Steps to Test This Pull Request
<!-- Steps to reproduce the behavior:
1. Run `cz bump -atm "a message"`
2. Run ` git tag -l --format='%(contents:subject)' <the-created-tag-name>`
3. Output should be `'a message'`


## Additional context
#558
